Runtime/Scripts/IDCCLauncher.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Unity.MeshSync
+{
+    public enum RunMode
+    {
+        GUI,
+        Background,
+        Console
+    }
+
+    /// <summary>
+    /// Interface for classes that handle DCC tool control from inside Unity.
+    /// </summary>
+    public interface IDCCLauncher
```

Comment:
Please make this into an internal class, unless we want to open this API to users.

Reply:
Done